control-plane/plugin/src/resources/tests.rs

```diff
@@ -340,25 +398,35 @@ where
     assert_eq!(&actual_output[..], expected_output);
 }
 
-fn pool_output(pool_state: PoolState) -> String {
+fn pool_output(pool_state: PoolState, managed: bool) -> String {
     let disks: String = pool_state.disks.join(", ");
+    let capacity = ::utils::bytes::into_human(pool_state.capacity);
+    let allocated = ::utils::bytes::into_human(pool_state.used);
+    let available = ::utils::bytes::into_human(pool_state.capacity - pool_state.used);
+    let committed = ::utils::bytes::into_human(pool_state.committed.unwrap_or_default());
     format!(
-        " {:id_width$}TOTAL CAPACITY  USED CAPACITY  {:disk_width$}{:node_width$}STATUS  MANAGED \n",
+        " {:id_width$}{:disk_width$}{:managed_width$}  {:node_width$}STATUS  {:capacity_width$}  {:allocated_width$}  {:available_width$}  {:committed_width$} \n",
         "ID",
         "DISKS",
+        "MANAGED",
         "NODE",
+        "CAPACITY",
+        "ALLOCATED",
+        "AVAILABLE",
+        "COMMITTED",
         id_width = pool_state.id.len() + 2,
         disk_width = disks.len() + 2,
-        node_width = pool_state.node.len() + 2
+        managed_width = managed.to_string().len() + 2,
+        node_width = pool_state.node.len() + 2,
+        capacity_width = capacity.len() + 2,
+        allocated_width = allocated.len() + 2,
+        available_width = available.len() + 2,
+        committed_width = committed.len() + 2
     ) + &*format!(
-        " {}  {}       {}        {}  {}  {}  {} \n",
-        pool_state.id,
-        pool_state.capacity,
-        pool_state.used,
-        disks,
-        pool_state.node,
-        pool_state.status.to_string(),
-        true
+        " {id}  {disks}  {managed}    {node}  {status}  {capacity}     {allocated}        {available}      {committed} \n",
+        id = pool_state.id,
+        node = pool_state.node,
+        status = pool_state.status.to_string(),
```

Comment:
Yeah agreed this is not a good way anymore now that the plugin has become more complex. The earlier intention was just to validate the stdout, tabular output, when the plugin was in initial stages.